### ISSUE TITLE
a11y

### ISSUE THREAD

**Issue by RDIL** [rdilweb/PickleCore#10]
We should have a way in the CSS to disable animations [cerca `prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) for the sidebars' transition.  

Note: because some browsers don't support it yet, we shouldn't wrap the animation definition in a media block, but rather have a block that sets the animation and related stuff to (essentially) `none`.
